### DISCUSSION TITLE
Range support for `Entry.getData`

### DISCUSSION THREAD

**Discussion by sntran** [gildas-lormeau/zip.js#443] [Q&A]
My use case is a zip file browser in which the index of the ZIP file are displayed and the user can select one file. If it's a MP4 file then it should be playable directly in the user's browser.

Right now I have an initial version on [Deno Deploy](https://dash.deno.com/playground/unzip) with a demo at https://unzip.deno.dev.

With the example, the MP4 is only played after all bytes are downloaded.

I would like to see if there is a way I can pass along the Range from the request headers and only retrieve data of the entry within that range.

**Comment by gildas-lormeau** [comment]
Isn't it the same question as https://github.com/gildas-lormeau/zip.js/discussions/444#discussioncomment-6728973?

**Reply by sntran** [reply]
No, this is different. Here I'm asking for a way to have an option to pass a byte range (i.e., `400-800`), and calling `getData` of an Entry will return me only the bytes within that range.
